src/Products/Signature/Controllers/SignatureApiController.cs

```diff
@@ -933,7 +959,7 @@ public HttpResponseMessage Sign(SignaturePostedDataEntity postedData)
         /// <param name="password">string</param>
         /// <param name="signaturesData">SignatureDataEntity</param>
         /// <param name="signsCollection">SignatureOptionsCollection</param>
-        private void SignDigital(string documentType, string password, SignatureDataEntity signaturesData, SignatureOptionsCollection signsCollection)
+        private void SignDigital(string documentType, string password, SignatureDataEntity signaturesData, List<SignOptions> signsCollection)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'SignDigital' a static method.](https://app.codacy.com/gh/groupdocs-signature/GroupDocs.Signature-for-.NET-MVC/pullRequest?prid=5325146)